core/domain/suggestion_registry.py

```diff
@@ -21,9 +23,20 @@
 
 (suggestion_models,) = models.Registry.import_models([models.NAMES.suggestion])
 
+ALL_CATEGORIES = ['Algebra', 'Algorithms', 'Architecture', 'Arithmetic',
```

Comment:
Is this copied from somewhere? Why are we repeating it here?

Reply:
It's copied from constants.js. Only the frontend defines this list of constants. So I copied it over. Not sure what else can be done.

Reply:
Ah -- in general, never copy. Always find a way to have just one source of truth. Otherwise someone's going to edit one thing and not the other. This is one of the most important principles in software development -- https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

constants.js is actually meant to be readable by both the frontend and the backend.  See constants.py.

Reply:
Oh ok! Thanks! I'll take a look 👍

Reply:
Removed copied list. Thanks!